sources/packages/web/src/components/common/ScholasticStandingForm.vue

```diff
@@ -5,14 +5,17 @@
     @submitted="submitted"
   >
     <template #actions="{ submit }">
-      <footer-buttons
-        :processing="processing"
-        @primaryClick="submit"
-        primaryLabel="Submit update"
-        @secondaryClick="cancel"
-        :disablePrimaryButton="readOnly"
-      /> </template
-  ></formio-container>
+      <div v-if="showCompleteInfo">
```

Comment:
Is the `div` required or the v-if can be moved inside the `footer-buttons` as below.
```vue
<footer-buttons
  v-if="showCompleteInfo"
  :processing="processing"
  @primaryClick="submit"
  primaryLabel="Submit update"
  @secondaryClick="cancel"
  :disablePrimaryButton="readOnly"
/>
```